### DISCUSSION TITLE
Is there any estimation about the next release availability?

### DISCUSSION THREAD

**Discussion by anatoliykim** [phpredis/phpredis#2272] [General]
Hi!

I was wondering if there are any plans about doing a new release any time soon?

The last release 5.3.7 seems to have been done quite a while ago, and there are several hundred commits done after it.

**Comment by yatsukhnenko** [comment]
We are going to prepare next release after resolving issue #2068

**Comment by michael-grunder** [comment]
Closing as we've had several releases
